doc/compiling/windows.md

```diff
@@ -1,68 +1,129 @@
 # Compiling on Windows using MSVC
 
-## Requirements
+In this guide, we use `path/to/minetest` to refer to the repository. Minetest was the old name of Luanti, and as of this writing, the repo has not been renamed.
 
-- [Visual Studio 2015 or newer](https://visualstudio.microsoft.com)
-- [CMake](https://cmake.org/download/)
-- [vcpkg](https://github.com/Microsoft/vcpkg)
-- [Git](https://git-scm.com/downloads)
+## 1. Machine requirements
 
+Luanti is compiled via Visual Studio and CMake, and version control is managed with Git.
 
-## Compiling and installing the dependencies
+-   [Visual Studio 2015 or newer](https://visualstudio.microsoft.com) including "Desktop development with C++" (includes CMake CLI but not CMake GUI)
+-   [Git](https://git-scm.com/downloads)
 
-It is highly recommended to use vcpkg as package manager.
+### 1.1. Visual Studio (VS) with C++
+
+1. Install VS from [visualstudio.microsoft.com](https://visualstudio.microsoft.com)
+1. In the VS installer, select "Desktop development with C++":
+
+    ![VS installer showing desktop development with C++ selected](assets/vs-installer.png)
+
+1. Confirm the installation.
```

Comment:
Why are all items numbered as 1?  It seems to be displayed properly as 1,2,3 in GitHub markdown preview but still.

Reply:
markdown has automatic numbering

Reply:
Yes, I use auto-numbering as it's easier to move or insert items later. Resolving